pkg/translator/testdata/default_collector_config.yaml

```diff
@@ -1,2 +1,9 @@
+processors:
+    usage:
+        honeycombextensionID: honeycomb
+extensions:
+    honeycomb:
+        opampextensionID: opamp
```

Comment:
A downside to this approach is that it produces a collector config that is dependent on the opampsupervisor adding an opampextension with the ID `opamp`.

Currently the opampsupervisor ALWAYS adds its own opampextension and it will replace any opampextension that the provided config already has. It always uses the ID `opamp` right now.

But it is possible the opampsupervisor changes in the future and that results in a (I think?) breaking change for HPSF from something outside our control.

Reply:
We have discussed recording compatibility version numbers on components before, eg works with Collector v0.120.0+. That could help with knowing what extension ID to use or other config for things outside of our control. 

If a pipeline had a target collector version, we could select the appropriate usage & extension component version with valid config.